CHANGELOG.md

```diff
@@ -46,7 +46,7 @@ A whole buncha bugfixes!
 * Added the `--signature-digest-algorithm` flag to `cosign verify`, allowing verification of container image signatures which were generated with a non-SHA256 signature algorithm (https://github.com/sigstore/cosign/pull/1071)
 * Builds should now be reproducible (https://github.com/sigstore/cosign/pull/1053)
 * Allows base64 files as `--cert` in `cosign verify-blob` (https://github.com/sigstore/cosign/pull/1088)
-* Kubernetes secrets generated for version >= 1.21 clusters have the immutible bit set (https://github.com/sigstore/cosign/pull/1091)
+* Kubernetes secrets generated for version >= 1.21 clusters have the immutable bit set (https://github.com/sigstore/cosign/pull/1091)
```

Comment:
Some projects don't like retroactive changes to release notes, let me know if you want me to drop all changes to this (or any other) file

Reply:
I don't see an issue with it!